lib/install/management/virtual_app.go

```diff
@@ -79,7 +79,7 @@ func (d *Dispatcher) createVApp(conf *config.VirtualContainerHostConfigSpec, set
 			Name:      "vSphere Integrated Containers",
 			Vendor:    "VMware",
 			VendorUrl: "http://www.vmware.com/",
-			Version:   "0.0.1",
+			Version:   "0.5.5",
```

Comment:
Using the `Version` var from the `version` pkg would be a better alternative to hardcoding the value. https://github.com/vmware/vic/blob/master/pkg/version/version.go#L25